playbooks/openstack-update_caasp.yml

```diff
@@ -0,0 +1,28 @@
+---
+
+- hosts: osh-deployer
+  gather_facts: no
+  tasks:
+    - name: Load variables
+      include_vars: "{{ item }}"
+      loop:
+        - "{{ playbook_dir }}/../vars/common-vars.yml"
+        - "{{ playbook_dir }}/../vars/manifest.yml"
+      tags:
+        - always
+
+    - name: Use Velum tooling to update CaaSP admin node
+      shell: "./velum-interactions --update-admin --update-minions -e {{ upstream_repos_clone_folder }}/ccp/environment.json"
```

Comment:
Do the `--update..` flags do the the `dup`? If so we wouldn't need to run it manually.